Core/UserDefaultsPropertyWrapper.swift

```diff
@@ -73,6 +73,7 @@ public struct UserDefaultsWrapper<T> {
         
         case emailWaitlistShouldReceiveNotifications = "com.duckduckgo.ios.showWaitlistNotification"
         case unseenDownloadsAvailable = "com.duckduckgo.app.unseenDownloadsAvailable"
+        case voiceSearch = "com.duckduckgo.app.voiceSearch"
```

Comment:
One tiny request here is that we rename this to `voiceSearchEnabled` to better match its intent (and also implies that this represents a boolean). Tbh as it is right now is fine, but if we add more voice search settings later it would help to avoid confusion.